ProvinceMapper/Source/Frames/Images/ImageCanvas.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <wx/wxprec.h>
+#ifndef WX_PRECOMP
+#include <wx/wx.h>
+#endif
+
+#include "../../Provinces/Province.h"
```

Comment:
You can set the ProvinceMapper/Source as an include directory and drop the ../..